man/nvim.1

```diff
@@ -373,8 +373,24 @@ Used to set the 'shell' option, which determines the shell used by the
 command.
 .It Ev NVIM_TUI_ENABLE_CURSOR_SHAPE
 If defined, change the cursor shape to a vertical bar while in insert mode.
-Requires that the host terminal support the DECSCUSR CSI escape sequence.
+Requires that the host terminal supports the DECSCUSR CSI escape sequence.
 Has no effect in GUIs.
+.Pp
+Depending on the terminal emulator, using this option with
```

Comment:
Here's the relevant passage from `man 7 mdoc`:

```
Nm
  The name of the manual page, or — in particular in section 1, 6, and 8
  pages — of an additional command or feature documented in the manual
  page.  When first invoked, the Nm macro expects a single argument, the
  name of the manual page.  Usually, the first invocation happens in the
  NAME section of the page.  The specified name will be remembered and used
  whenever the macro is called again without arguments later in the page.
 ...
```

Here's what the sentence looks like after being formatted:
```
                 Depending on the terminal emulator, using this option with
                 nvim under tmux(1) might require adding the following to
                 ~/.tmux.conf:
```